File: src/main/my-app/src/domain/FirstLogin.js
export class FirstLogin {
    _userId = null;
    _password = null;

    constructor(data) {
        if (!data) {
            return;
        }
        if (data.hasOwnProperty("userId")) {
            this._userId = data.userId;
        }
        if (data.hasOwnProperty("password")) {
            this._password = data.password;
        }
    }

    get userId() {
        return this._userId;
    }

    set userId(value) {
        this._userId = value;
    }

    get password() {
        return this._password;
    }

    set password(value) {
        this._password = value;
    }

    get requestObject() {
        return {
            userId: this.userId,
            password: this.password
        }
    }
}

File: src/main/java/com/example/itfinalproject/domain/FirstLogin.java
package com.example.itfinalproject.domain;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Entity
@NoArgsConstructor
@Table(name = "first_login")
public class FirstLogin {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    private Long userId;

    private String password;

}
